src/consumables/flavor/models.py

```diff
@@ -0,0 +1,16 @@
+from ..models import BaseConsumable
+
+
+class Flavor(BaseConsumable):
+    name: str
+    flavor_type: str
+
+    @classmethod
+    def create(cls, *args, **kwargs):
```

Comment:
Add type hinting for create method, for example:
`def create(cls, *args, **kwargs) -> 'Flavor'`